client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from 'react';
+import { AppBar, Toolbar, Avatar, Button, Box } from '@material-ui/core';
+import Menu from '@material-ui/core/Menu';
+import { useAuth } from '../../context/useAuthContext';
+import logo from '../../Images/logo.png';
+import profile from '../../Images/profile.png';
+import useStyles from './useStyles';
+import { useHistory } from 'react-router-dom';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+
+const Navbar = (): JSX.Element => {
+  const { logout } = useAuth();
+  const classes = useStyles();
+  const history = useHistory();
+  const { location } = history;
+  const showAuthButtons = location.pathname == '/login' || location.pathname == '/signup' ? true : false;
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <AppBar position="static" className={classes.root}>
+      <Toolbar>
+        <Box className={classes.box}>
+          <img src={logo} alt="logo" className={classes.logo} />
+        </Box>
+
+        {!showAuthButtons && (
+          <Box className={classes.navButtons} justifyContent="space-between" alignItems="center">
+            <Button
+              size="large"
+              onClick={() => {
+                history.push('/discover');
+              }}
+              color="primary"
+              className={classes.button}
+            >
+              Discover
+            </Button>
+            <Button
+              size="large"
+              onClick={() => {
+                history.push('/messages');
+              }}
+              color="primary"
+              className={classes.button}
+            >
+              Messages
+            </Button>
+            <Button
+              size="large"
+              onClick={() => {
+                history.push('/notifications');
+              }}
+              color="primary"
+              className={classes.button}
+            >
+              Notifications
+            </Button>
+          </Box>
+        )}
+
+        <Box className={classes.navButtons} justifyContent="space-between" alignItems="center">
+          {location.pathname == '/login' && (
+            <Button
+              size="large"
+              onClick={() => {
+                history.push('/signup');
```

Comment:
We don't need two Buttons here, once you have refactored the code to use `Link`, you can just use a ternary operator to conditionally route to a different page.

Reply:
ah yes, I changed it, thanks